README.md

```diff
@@ -1,58 +1,42 @@
 
 ## Zeppelin API client plugin for Drone
 
-Zeppelin REST API client plugin for Drone. A step in the Pipeline PaaS CI/CD component to provision a Kubernetes cluster or use a managed one.
+Zeppelin REST API client plugin for Drone. A step in the Pipeline PaaS CI/CD component to create / update / delete and also run a Zeppelin notebook via Zeppelin's REST API.
+You had to specify desired state of Zeppelin notebook in zeppelin_notebook_state option.
+These are the valid states:
 
-### Example drone config
+- present - notebook will created or updated
+- absent - notebook will be deleted if exists
+- running - notebook will be created or updated and started
 
-.drone.yml
+#### Specify required secrets
 
-    pipeline:
-      cluster:
-        image: banzaicloud/zeppelin-client:latest
+Provide valid credentials for the pipeline API.
 
-        log_level: info
-        log_format: text
+These options needs to be specified in the CI/CD [GUI](https://github.com/banzaicloud/pipeline/blob/master/docs/pipeline-howto.md#cicd-secrets).
 
-        cluster_name: "demo-cluster1"
-        cluster_location: "eu-west-1"
-        cluster_state: "created"
+* plugin_zeppelin_username: Zeppelin username
```

Comment:
Please list all the secrets here otherwise for a newcomer it will be difficult to figure out what other secrets needs to be set beside these too.